package.json

```diff
@@ -55,6 +55,8 @@
     "mocha-text-cov": "^0.1.0",
     "nock": "^1.2.1",
     "read": "^1.0.5",
-    "time-grunt": "^1.1.0"
+    "time-grunt": "^1.1.0",
+    "mockery": "^1.4.0",
+    "sinon": "^1.15.4"
```

Comment:
Small nit, but looks previous authors have tried to keep the list in alphabetical order.